packages/mobile/src/redux/migrations.ts

```diff
@@ -409,4 +409,11 @@ export const migrations = {
       isSettingPin: undefined,
     },
   }),
+  32: (state: any) => ({
```

Comment:
not sure we need v32 just yet, I don't see a release branch cut for v31 yesterday... just the hotfix which doesn't include any schema migration

Reply:
also, I'm not sure we need this to begin with, since the migration will produce a falsy value for hasLinkedBankAccount anyway

Reply:
Asked in [#engineering-team](https://valora-app.slack.com/archives/C025V1D6F3J/p1643913042569519)

Apparently we should update it every time!